src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -50,11 +50,11 @@ public partial class Socket : IDisposable
         // Our internal state doesn't automatically get updated after a non-blocking connect
         // completes.  Keep track of whether we're doing a non-blocking connect, and make sure
         // to poll for the real state until we're done connecting.
-        private bool _nonBlockingConnectInProgress;
+        private bool _pollPendingConnect;
 
         // Keep track of the kind of endpoint used to do a non-blocking connect, so we can set
         // it to _rightEndPoint when we discover we're connected.
-        private EndPoint? _nonBlockingConnectRightEndPoint;
+        private EndPoint? _pendingConnectRightEndPoint;
```

Comment:
Since this is now used in more cases than just non-blocking connect, we should update the comment.